backend/app/api/endpoints/templates.py

```
@@ -32,12 +34,16 @@ def get_template_by_id(template_id: str, db: Session = Depends(get_db)):
     if not template_data.get("category"):
-        raise HTTPException(status_code=400, detail="Template must have a category")
+        raise HTTPException(
+            status_code=400, detail="Template must have a category"
+        )
     return template_service.create_template(template_data)
 
 
 @router.put("/{template_id}", response_model=TemplateResponse)
-def update_template(template_id: str, updates: Dict, db: Session = Depends(get_db)):
+def update_template(
+    template_id: str, updates: Dict, db: Session = Depends(get_db)
```

Ruff (0.12.2)

Unused function argument: `db`

(ARG001)